notify failing on MacOS due to new fsevent-sys update

On the [bevy](https://github.com/bevyengine/bevy) github page, our CI builds are failing due to a recent update within the `fsevent-sys` crate.
See CI failures [here](https://github.com/bevyengine/bevy/pull/2149/checks?check_run_id=2567791064).

CI failed due to: 
```
error[E0308]: mismatched types
   --> /Users/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/notify-5.0.0-pre.7/src/fsevent.rs:361:17
    |
361 |                 callback,
    |                 ^^^^^^^^ expected `u32`, found enum `c_void`
    |
    = note: expected fn pointer `extern "C" fn(_, _, _, _, *const u32, *const u64)`
                  found fn item `extern "C" fn(_, _, _, _, *const c_void, *const c_void) {callback}`
```

Here is the related `notify` [issue](https://github.com/notify-rs/notify/issues/314)

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
 
 name = "fsevent"
-version = "2.1.0"
+version = "2.1.1"
 authors = [ "Pierre Baillet <pierre@baillet.name>" ]
 description = "Rust bindings to the fsevent-sys macOS API for file changes notifications"
 license = "MIT"
@@ -10,7 +10,7 @@ edition = "2018"
 
 [dependencies]
 bitflags = "1"
-fsevent-sys = "3.2.0"
+fsevent-sys = "4.0.0"
 # fsevent-sys = { path = "fsevent-sys" }
 
 [dev-dependencies]

--- a/fsevent-sys/Cargo.toml
+++ b/fsevent-sys/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "fsevent-sys"
-version = "3.2.0"
+version = "4.0.0"
 authors = ["Pierre Baillet <pierre@baillet.name>"]
 description = "Rust bindings to the fsevent macOS API for file changes notifications"
 license = "MIT"

